src/Draco.Editor.Web/app/ts/app.ts

```diff
@@ -132,16 +123,48 @@ const outputEditor = monaco.editor.create(document.getElementById('output-viewer
     readOnly: true
 });
 
+worker.onmessage = (ev) => {
+    const msg = ev.data as {
+        type: string;
+        message: string;
+    };
+    switch (msg.type) {
+    case 'setOutputText':
+        outputEditor.getModel().setValue(msg.message);
+        break;
+    default:
+        break;
+    }
+};
+
+
```

Comment:
Double bank line, maybe do autoformat?

Reply:
autoformat does not remove blanks lines.